sdks/java/core/src/test/java/org/apache/beam/sdk/util/ReduceFnRunnerTest.java

```diff
@@ -604,8 +630,8 @@ public void testPaneInfoFinalAndOnTime() throws Exception {
     ReduceFnTester<Integer, Iterable<Integer>, IntervalWindow> tester = ReduceFnTester.nonCombining(
         WindowingStrategy.of(FixedWindows.of(Duration.millis(10)))
             .withTrigger(
-                Repeatedly.<IntervalWindow>forever(AfterPane.<IntervalWindow>elementCountAtLeast(2))
-                    .orFinally(AfterWatermark.<IntervalWindow>pastEndOfWindow()))
+                Repeatedly.<IntervalWindow>forever(AfterPane.elementCountAtLeast(2))
```

Comment:
As long as you are cleaning up unused generic params, might as well get this one.